rules.html

```diff
@@ -122,10 +122,26 @@ <h2>2. Behavior and chat</h2>
             </li>
             <li>
                 <b>2.4 Toxicity.</b><br/>
-                Toxicity is not allowed on this server. Toxicity in chat can be for instance using racial slurs, sexual harassment or severely explicit language.
-                It can also be other non-chat actions which drive players away from the community, for instance portal trapping, luring newly joined players to your town so you can kill them or lock them in your town jail.<br/>
+                Toxicity is not allowed on this server. Toxicity in chat includes but is not limited to:<br/>
+                <ul>
+                    <li><b>2.4.a Harassment:</b> This includes but not limited to:</li>
+                    <li>
+                        <ul class="sub_list">

```

Comment:
I don't think there should be an inner list here. The outer list only has two nodes, the inner list and one extra list item added on at the end, doesn't really make much sense structurally. They should be moved out to the outer list.

Reply:
Test!

Reply:
Okay, wasn't letting me comment here before.

This is the same design you go with if you have multiple sub-menus in a dropdown navigation menu, and this is a more organized way than putting `<br/>` after every line.

Reply:
As there are two sub-lists under `2.4`, which are 
As both `2.4.a` and `2.4.b` as lists, they are both in their list under the main list `2.4`. And as `2.4.a` have a list of itself too, it has another list for that.